modules/localgov_workflows_notifications/src/NotificationTimerInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\localgov_workflows_notifications;
+
+/**
+ * Interface for handling notification timing.
+ */
+interface NotificationTimerInterface {
+
+  /**
+   * Last notification state variable name.
+   *
+   * @var string
+   */
+  const LAST_RUN = 'localgov_workflows_notifications.last_email_run';
```

Comment:
Ohhh... this is a nit-pick. So much so I don't even suggest you change it. But isn't this an implementation detail rather than something to be shared in the interface.